src/m365/viva/commands/engage/engage-community-list.ts

```diff
@@ -0,0 +1,35 @@
+import { Logger } from '../../../../cli/Logger.js';
+import { odata } from '../../../../utils/odata.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+import { Community } from './Community.js';
+
+class VivaEngageCommunityListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ENGAGE_COMMUNITY_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists Viva Engage communities';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'privacy'];
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    if (this.verbose) {
+      await logger.logToStderr('Getting the list of Viva Engage communities');
+    }
+
+    try {
+      const results = await odata.getAllItems<Community>(`${this.resource}/v1.0/employeeExperience/communities`);
```

Comment:
Seems like we forgot to add the `Community.ReadWrite` scope for the previous commands in the allScopes parameter. Could you add it in the `src/config.ts`, please?

Reply:
Rebased with the latest master and scope `Community.ReadWrite.All` added